dolphin/alert_manager/trap_receiver.py

```diff
@@ -12,19 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from oslo_log import log
+import re
 
-from pysnmp.entity import engine, config
+from oslo_log import log
 from pysnmp.carrier.asyncore.dgram import udp
+from pysnmp.entity import engine, config
 from pysnmp.entity.rfc3413 import ntfrcv
 from pysnmp.proto.api import v2c
-from pysnmp.smi import builder, view, rfc1902, error
+from pysnmp.smi import builder, view, rfc1902
 
+from dolphin import exception
+from dolphin.alert_manager import alert_processor
 from dolphin.alert_manager import constants
 
 LOG = log.getLogger(__name__)
 
-# Currently static mib file list is loaded, logic to be changed to load all mib file
+# Currently static mib file list is loaded,
+# logic to be changed to load all mib file
 MIB_LOAD_LIST = ['SNMPv2-MIB','IF_MIB']
 
```

Comment:
Seems this file is not formatted, need two lines here.

Reply:
Updated